src/main/java/com/demo/classroom/Utility/Constants.java

```diff
@@ -7,11 +7,15 @@ public enum Constants {
     EMAIL_TAKEN("Email is already taken."),
     
     TEACHER_REG_SUCCESSFULL("Teacher registered successfully"),
+    TEACHER_LOGIN_SUCCESSFULL("Teacher login successful"),
```

Comment:
Check the spelling of other constants as well
```suggestion
    TEACHER_LOGIN_SUCCESSFUL("Teacher login successful"),
```

Reply:
Done